test/sdk-commands/utils/data-layer-fs.spec.ts

```diff
@@ -0,0 +1,100 @@
+import path from 'path'
+import {
+  createFileSystemInterfaceFromFsComponent,
+  pathToPosix
+} from '../../../packages/@dcl/sdk-commands/src/commands/start/data-layer/fs'
+import { createFsComponent } from '../../../packages/@dcl/sdk-commands/src/components/fs'
+
+describe('pathToPosix', () => {
+  test('should convert Windows-style paths to POSIX-style paths', () => {
+    const input = 'scene\\assets\\main.composite'
+    const expectedOutput = 'scene/assets/main.composite'
+    expect(pathToPosix(input)).toEqual(expectedOutput)
+  })
+
+  test('should keep POSIX-style paths unchanged', () => {
+    const input = 'scene/assets/main.composite'
+    const expectedOutput = 'scene/assets/main.composite'
+    expect(pathToPosix(input)).toEqual(expectedOutput)
+  })
+})
+
+describe('FsInterface', () => {
+  const fs = createFsComponent()
+  const fsInterface = createFileSystemInterfaceFromFsComponent({ fs })
+
+  describe('dirname', () => {
+    test('should return the directory name of the given path', () => {
+      const input = 'scene/assets/main.composite'
+      const expectedOutput = 'scene/assets'
+      expect(fsInterface.dirname(input)).toEqual(expectedOutput)
+    })
+
+    test('[Win] should return the directory name of the given path', () => {
+      const input = 'scene\\assets\\main.composite'
+      const expectedOutput = 'scene/assets'
+      const dirname = path.dirname
+      path.dirname = path.win32.dirname
+      expect(fsInterface.dirname(input)).toEqual(expectedOutput)
+      path.dirname = dirname
+    })
+
+    test('should return an empty string if the path is a file name', () => {
```

Comment:
nip tick here, but it should be a `"."` rather than an empty string no?

Reply:
yes !